src/Propel/Generator/Behavior/Versionable/VersionableBehaviorObjectBuilderModifier.php

```diff
@@ -330,11 +365,11 @@ protected function addToVersion(&$script)
  * Sets the properties of the current object to the value they had at a specific version
  *
  * @param   integer \$versionNumber The version number to read
- * @param   ConnectionInterface \$con The connection to use
+ * @param   ConnectionInterface \$con (optional) The ConnectionInterface connection to use.
```

Comment:
Can you please remove everywhere "(optional)"? We haven't used that in the code.